stackable-base/Dockerfile

```diff
@@ -156,8 +159,8 @@ if [ "$(trust list --filter=ca-anchors | grep -c 'E-Tugra')" != "0" ]; then
 fi
 EOF
 
-COPY --from=product-utils-builder --chown=stackable:stackable /config-utils/target/release/config-utils /stackable/config-utils
-COPY --from=product-utils-builder --chown=stackable:stackable /config-utils/config-utils.cdx.xml /stackable/config-utils.cdx.xml
+COPY --from=product-utils-builder --chown=${STACKABLE_USER_UID}:0 /config-utils/target/release/config-utils /stackable/config-utils
```

Comment:
If I understand the linked docs correctly, the group permissions must also be the same as the user permissions. This might be implicit here (I don't know) but we might want to also be explicit about it.

Reply:
Yeah, we set the group to `0` as per the recommendation (and it makes sense as well).
I'm not sure if I understand you Razvan. Could you rephrase your comment for me?

Reply:
I mean the actual permissions have to be the same for both entities, not 755 for user and 000 for the group (for example)

Reply:
That is what this line should be doing

```
chmod -R g=u /stackable
```

https://github.com/stackabletech/docker-images/pull/849/files#diff-d27bbada86d18407f7107257ec59dcaf6375b084613d33ab71d3499c3fa84441R113

Would this cover your concern?

Reply:
Yes, I hadn't seen that.